src/main/asyncapi/network_survey_messaging.yaml

```diff
@@ -1031,6 +1060,218 @@ components:
+            rawMessage:
+              $ref: '#/components/schemas/rawMessage'
+
+
+    umtsNasDsdsPayload:
+      type: object
+      required:
+        - version
+        - messageType
+        - data
+      properties:
+        version:
+          $ref: '#/components/schemas/version'
+        messageType:
+          $ref: '#/components/schemas/messageType'
+          description: The type of message, must be UmtsNasDsds.
+          example: "UmtsNasDsds"
+
+        data:
+          type: object
+          description: The payload of this message that contains all the message data.
+          required:
+            - deviceSerialNumber
+            - deviceName
+            - deviceTime
+            - missionId
+            - channelType
+            - rawMessage
+          properties:
+            deviceSerialNumber:
+              $ref: '#/components/schemas/deviceSerialNumber'
+            deviceName:
+              $ref: '#/components/schemas/deviceName'
+            deviceTime:
+              $ref: '#/components/schemas/deviceTime'
+            latitude:
+              $ref: '#/components/schemas/latitude'
+            longitude:
+              $ref: '#/components/schemas/longitude'
+            altitude:
+              $ref: '#/components/schemas/altitude'
+            missionId:
+              $ref: '#/components/schemas/missionId'
+
+            channelType:
+              $ref: '#/components/schemas/channelType'
+              example: "PLAIN"
+              enum:
+                - "UNKNOWN"
+                - "PLAIN"
+                - "SEC_HEADER"
+            rawMessage:
+              $ref: '#/components/schemas/rawMessage'
+
+
+    umtsNasPayload:
+      type: object
+      required:
+        - version
+        - messageType
+        - data
+      properties:
+        version:
+          $ref: '#/components/schemas/version'
+        messageType:
+          $ref: '#/components/schemas/messageType'
+          description: The type of message, must be UmtsNas.
+          example: "UmtsNas"
+
+        data:
+          type: object
+          description: The payload of this message that contains all the message data.
+          required:
+            - deviceSerialNumber
+            - deviceName
+            - deviceTime
+            - missionId
+            - channelType
+            - rawMessage
+          properties:
+            deviceSerialNumber:
+              $ref: '#/components/schemas/deviceSerialNumber'
+            deviceName:
+              $ref: '#/components/schemas/deviceName'
+            deviceTime:
+              $ref: '#/components/schemas/deviceTime'
+            latitude:
+              $ref: '#/components/schemas/latitude'
+            longitude:
+              $ref: '#/components/schemas/longitude'
+            altitude:
+              $ref: '#/components/schemas/altitude'
+            missionId:
+              $ref: '#/components/schemas/missionId'
+
+            channelType:
```

Comment:
The channelType does not apply to UmtsNas like it does for LteNas, so this field can be removed.

Reply:
Removed the field, classes and calls to umtsnaschanneltype/umtsnasdsdschanneltype.